<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DifNum</class>
 <widget class="QMainWindow" name="DifNum">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DifNum</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="Tablain">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>130</width>
      <height>355</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>X-5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>X-4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>X-3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>X-2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>X-1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>X0</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>X1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>X2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>X3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>X4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>X5</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>F(X)</string>
     </property>
    </column>
    <item row="5" column="0">
     <property name="background">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>127</blue>
       </color>
      </brush>
     </property>
    </item>
   </widget>
   <widget class="QDoubleSpinBox" name="Incremento">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>0</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Carga">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cargar Puntos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Inicia">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>120</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Iniciar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Limpia">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>190</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Sale">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>260</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="TablaAtras">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>295</width>
      <height>265</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>f'(x) | O(h)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>f'(x) | O(h^2)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>f''(x) | O(h)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>f''(x) | O(h^2)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>f'''(x) | O(h)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>f'''(x) | O(h^2)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>f''''(x) | O(h)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>f''''(x) | O(h^2)</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Solucion</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Error (%)</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>211</red>
        <green>211</green>
        <blue>211</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>211</red>
        <green>211</green>
        <blue>211</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>195</red>
        <green>205</green>
        <blue>225</blue>
       </color>
      </brush>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>255</red>
        <green>156</green>
        <blue>158</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>211</red>
        <green>211</green>
        <blue>211</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>211</red>
        <green>211</green>
        <blue>211</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>211</red>
        <green>211</green>
        <blue>211</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>211</red>
        <green>211</green>
        <blue>211</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="6" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>211</red>
        <green>211</green>
        <blue>211</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="6" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>211</red>
        <green>211</green>
        <blue>211</blue>
       </color>
      </brush>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="TablaCentro">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>420</y>
      <width>295</width>
      <height>265</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>f'(x) | O(h^2)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>f'(x) | O(h^4)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>f''(x) | O(h^2)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>f''(x) | O(h^4)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>f'''(x) | O(h^2)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>f'''(x) | O(h^4)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>f''''(x) | O(h^2)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>f''''(x) | O(h^4)</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Solucion</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Error (%)</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>211</red>
        <green>211</green>
        <blue>211</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>211</red>
        <green>211</green>
        <blue>211</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>211</red>
        <green>211</green>
        <blue>211</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>211</red>
        <green>211</green>
        <blue>211</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>211</red>
        <green>211</green>
        <blue>211</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>211</red>
        <green>211</green>
        <blue>211</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="6" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>211</red>
        <green>211</green>
        <blue>211</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="6" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>211</red>
        <green>211</green>
        <blue>211</blue>
       </color>
      </brush>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="TablaAdelante">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>420</y>
      <width>299</width>
      <height>265</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>f'(x) | (O(h)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>f'(x) | (O(h^2)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>f''(x) | (O(h)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>f''(x) | (O(h^2)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>f'''(x) | (O(h)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>f'''(x) | (O(h^2)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>f''''(x) | (O(h)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>f''''(x) | (O(h^2)</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Solucion</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Error (%)</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>211</red>
        <green>211</green>
        <blue>211</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>211</red>
        <green>211</green>
        <blue>211</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>211</red>
        <green>211</green>
        <blue>211</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>211</red>
        <green>211</green>
        <blue>211</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>211</red>
        <green>211</green>
        <blue>211</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>211</red>
        <green>211</green>
        <blue>211</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="6" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>211</red>
        <green>211</green>
        <blue>211</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="6" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>211</red>
        <green>211</green>
        <blue>211</blue>
       </color>
      </brush>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>despues ingrese cada punto en</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Incremento:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>390</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Derivada Hacia Atras</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>390</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Derivada Centralizada</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>390</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Derivada Hacia Adelante</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Casos">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Seleccionar</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Seleccione un numero de puntos,</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>la posicion correspondiente.</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>Sale</sender>
   <signal>clicked()</signal>
   <receiver>DifNum</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>604</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>701</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
